apps/server/src/modules/files/job/sync-files.console.ts

```diff
@@ -0,0 +1,35 @@
+/* istanbul ignore file */
+
+import { Logger } from '@src/core/logger';
+import { Command, Console } from 'nestjs-console';
+import { SyncFilesUc } from '../uc/sync-files.uc';
+
+@Console({ command: 'sync-files' })
+export class SyncFilesConsole {
+	constructor(private syncFilesUc: SyncFilesUc, private logger: Logger) {
+		this.logger.setContext(SyncFilesConsole.name);
+	}
+
+	@Command({ command: 'tasks [aggregationSize] [numParallelPromises]' })
+	async syncFilesForTasks(aggregationSize = 5000, numParallelPromises = 50) {
+		let itemsFound: number;
+		let aggregationsCounter = 0;
+
+		do {
+			this.logger.log(
+				`Starting file sync: aggregation #${
+					aggregationsCounter + 1
+				} with aggregationSize = ${aggregationSize} and numParallelPromises = ${numParallelPromises}`
+			);
+			// eslint-disable-next-line no-await-in-loop
+			itemsFound = await this.syncFilesUc.syncFilesForTasks(
+				{
+					aggregationSize: Number(aggregationSize),
+					numParallelPromises: Number(numParallelPromises),
+				},
+				{ fileCounter: aggregationsCounter * Number(aggregationSize) }
+			);
+			aggregationsCounter += 1;
+		} while (itemsFound > 0);
```

Comment:
Why not put this to a uc function. I think the function are much more readable if any this.logger.log( is put to a additional private function like this.logStartFileSync(aggregationsCounter); For understanding functions is is not usefull to see the bigger logging blocks.